docs/ballerina-by-example/examples/async/async.bal

```diff
@@ -10,7 +10,8 @@ endpoint http:ClientEndpoint clientEndpoint { targets:[{url: "https://postman-ec
 function main(string[] args) {
   // call the function "sum" asynchronously
   future<int> f1 = async sum(40, 50);
-  // future values can be passed around to get the result later
+  // Pass the value of the variable 'f1' of type future to 
```

Comment:
This doesn't capture the meaning of 'future' can be passed around and you can get the results later.